src/screens/LoginScreen.js

```diff
@@ -1,7 +1,7 @@
 // 3rd part libraries - core
 import {AppLoading} from 'expo';
 import React, {Component} from 'react';
-import {View, Text, AsyncStorage} from 'react-native';
```

Comment:
We were previously explicitly calling `AsyncStorage` here in `LoginScreen` to see if a token already existed. Now with `redux-persist`, if it exists, it'll be part of the props from the start.